plugins/mcreator-core/procedures/entity_get_uuid.json

```diff
@@ -0,0 +1,20 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "output": "String",
```

Comment:
The method you're calling returns a UUID, not a String. This will cause compilation errors when used with text procedures. I'd suggest removing this block, as the "Get UUID as string" can be used instead

Reply:
Done